flight-icons/README.md

```diff
@@ -0,0 +1,33 @@
+# flight-icons
+
+An addon for the icons from `flight`. Currently in development. This addon can be used in React or Ember apps.
+
+## Installation
+
+```bash
+yarn add @hashicorp/flight-icons
+```
+
+## Usage
+
+### How to choose 16 or 24 viewBox size
```

Comment:
I am not sure this section is accurate. I think the viewbox always stays the same but the height and width can change. See [this CodePen](https://codepen.io/melsumner/pen/1143304b4933a757d0a1e8233881bba7?editors=1100) and also [this PR](https://github.com/hashicorp/flight/pull/126).

Reply:
yeap - not sure why you did request changes, this is a draft pr so just looking for discussion. to start it off, i made the pr to collect feedback from the team

Reply:
added this PR to the sync notes for tomorrow. not crosslinking here cause internal link